manifests/base/files/gen-istio-cluster.yaml

```diff
@@ -6722,7 +6722,7 @@ spec:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: istiod-istio-system
+  name: istiod-default
```

Comment:
Not introduced by your PR, but clusterrole shouldn't have any namespace at all, right?

Reply:
It does - so you can install multi-tenant control planes in different namespaces.